app/models/question.rb

```diff
@@ -36,14 +36,18 @@ def starred_by?(user)
     evaluation_by(:stars, user) == 1
   end
 
+  def expired?
```

Comment:
So add a new column 'closed' and run a cron job nightly to update questions that are older than 30 days?